FPDocumentView.m

```diff
@@ -177,8 +177,15 @@ - (void)scrollToMidpointOnPage:(unsigned int)page point:(NSPoint)midPoint
     NSPoint viewPoint = [self convertPoint:midPoint fromPage:page];
     NSPoint viewOrigin = NSMakePoint(floorf(viewPoint.x - viewWidth/2.0),
                                      floorf(viewPoint.y - viewHeight/2.0));
+/*
+ TODO
+ Fix the error:
+ Sending 'NSPoint' (aka 'struct CGPoint') to parameter of incompatible type 'NSRect' (aka 'struct CGRect')
     [[_scrollView contentView] scrollToPoint:
         [[_scrollView contentView] constrainBoundsRect:viewOrigin]];
+ */
+    [[_scrollView contentView] scrollToPoint:viewOrigin];
+
```

Comment:
What is left to do for this TODO? Would it be possible for you to do the fix?

Reply:
I really don't know what I did here ;-)
Once I removed the `constrainBoundsRect` I get rid of the error but what are the implication? No idea.